src/store/integrations/composer.tsx

```diff
@@ -256,8 +257,7 @@ const Composer: FC<ComposerProps> = ({
 		<Container
 			height="100%"
 			crossAlignment="baseline"
-			mainAlignment="flex-start"
-			style={{ overflowY: 'hidden' }}
+			style={{ overflowY: 'hidden', display: 'initial' }}
```

Comment:
Thanks, that's a nice suggestion. I tried to make as less changes as possible to avoid redefining or using a new component, not knowing the impact it would have on other components. I'll discuss it with my team

Reply:
I reverted these changes and replaced the quicklink with link